tests/test_http_api.py

```diff
@@ -26,3 +26,14 @@ def test_httpbin_ip():
      else:
          print('Долгий ответ сервера :( = ', r.elapsed.total_seconds(), "\n")
 
+
+def test_httpbin_user_agent():
+    r = requests.get(base_url + 'user-agent')
+    assert r.status_code == 200, r.text
+    assert r.reason == "OK", r.text
+    print(r.elapsed.total_seconds())
+    #data = r.json()
+    assert r.json()['user-agent'] == 'python-requests/' + requests.__version__
+    #assert len(r.json())
+    #print(data)
+    #print(r.reason)
```

Comment:
pretty good!
just need this in a new repository now :)
https://github.com/testlikeachamp/api_tests